features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx

```diff
@@ -0,0 +1,585 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { Link } from 'react-router-dom';
+import Joi from '@hapi/joi';
+import { withStyles } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Checkbox from '@material-ui/core/Checkbox';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Grid from '@material-ui/core/Grid';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import DescriptionIcon from '@material-ui/icons/Description';
+import ContentBase from 'AppComponents/AdminPages/Addons/ContentBase';
+import HelpBase from 'AppComponents/AdminPages/Addons/HelpBase';
+import Alert from 'AppComponents/Shared/Alert';
+import API from 'AppData/api';
+import Configurations from 'Config';
+import ChipInput from 'material-ui-chip-input';
+import PropTypes from 'prop-types';
+
+const styles = (theme) => ({
+    alertsWrapper: {
+        padding: theme.spacing(2),
+        '& span, & h5, & label, & input, & td, & li': {
+            color: theme.palette.getContrastText(theme.palette.background.paper),
+        },
+    },
+    manageAlertHeading: {
+        marginBottom: theme.spacing(),
+    },
+    chipInput: {
+        width: '100%',
+        marginTop: theme.spacing(2),
+        marginBottom: theme.spacing(2),
+    },
+    alertConfigDialog: {
+        width: '60%',
+    },
+    configDialogHeading: {
+        fontWeight: '600',
+    },
+    btnContainer: {
+        marginTop: theme.spacing(),
```

Comment:
Shall we use `[<Box mt={1}>](https://material-ui.com/components/box/#box)` to do this ?

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8662/commits/ad2be95172c14f083670b0910b54a6435b9f97bc